integration/tests/basic/rpc/test_rpc_base_calls.py

```diff
@@ -288,13 +322,15 @@ def test_eth_get_storage_at_eq_val(self, json_rpc_client):
         assert "error" not in response
         assert "result" in response
         assert new_data in web3.Web3.to_text(response["result"]), "wrong variable value"
+        EthGetStorageAt(**response)
 
-    @pytest.mark.mainnet
+    @pytest.mark.neon_only
```

Comment:
mainet mark has other porpose, please don't remove it

Reply:
fixed ✅